tests/security/dns_cert/scripts/dns_cert.txt

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# Copyright Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -u
+set -o pipefail
+
+source ${REPO_ROOT}/content/en/docs/tasks/security/dns-cert/snips.sh
+
+out=$(snip_check_the_provisioning_of_dns_certificates_1 2>&1)
+# Remove trailing spaces
+out=$(echo "$out" | sed 's/[ ]*$//')
+_verify_contains "$out" "$snip_check_the_provisioning_of_dns_certificates_2" "snip_check_the_provisioning_of_dns_certificates_1"
+
+snip_regenerating_a_dns_certificate_1
+
+out=$(snip_regenerating_a_dns_certificate_2 2>&1)
+# Remove trailing spaces
+out=$(echo "$out" | sed 's/[ ]*$//')
```

Comment:
```suggestion
```

Reply:
This line (i.e., removing the trailing spaces in $out) is necessary and the test fails if this line is removed.